src/components/AppLayout/ContextualLinks/useContextualLink.tsx

```diff
@@ -0,0 +1,95 @@
+import { useAnalytics } from "@dashboard/components/ProductAnalytics/useAnalytics";
+import {
+  API_GUIDE_DOCS,
+  API_REFERENCE_DOCS,
+  EXTENDING_WITH_WEBHOOKS_DOCS_URL,
+  ORDER_MANAGEMENT_DOCS_URL,
+  PRODUCT_CONFIGURATION_DOCS_URL,
+  USER_PERMISSIONS_DOCS_URL,
+} from "@dashboard/links";
+import * as React from "react";
+import { useIntl } from "react-intl";
+
+import { ContextualLine } from "./ContextualLine";
+import { contextualLinks } from "./messages";
+
+type SubtitleType =
+  | "extending_saleor"
+  | "product_list"
+  | "order_list"
+  | "dev_panel"
+  | "webhooks_events"
+  | "staff_members";
+
+const EVENT = "contextual_link_clicked";
+
+export const useContextualLink = (type: SubtitleType) => {
```

Comment:
it does not really make sense to wrap it and keep centralised, these are static info that simply could be places within the corresponding page

Reply:
My reasoning was that in each place where I would put that info I would have to import intl, messages, analytics and link component. I think keeping it centralised doesn't make the code bloated.